explain-aggregation.md

```diff
@@ -0,0 +1,172 @@
+
+{{< copyable "sql" >}}
+
+```sql
+SHOW TABLE t1 REGIONS;
+```
+
+```sql
++-----------+--------------+--------------+-----------+-----------------+-------+------------+---------------+------------+----------------------+------------------+
+| REGION_ID | START_KEY    | END_KEY      | LEADER_ID | LEADER_STORE_ID | PEERS | SCATTERING | WRITTEN_BYTES | READ_BYTES | APPROXIMATE_SIZE(MB) | APPROXIMATE_KEYS |
++-----------+--------------+--------------+-----------+-----------------+-------+------------+---------------+------------+----------------------+------------------+
+|        64 | t_64_        | t_64_r_31766 |        65 |               1 | 65    |          0 |          1325 |  102033520 |                   98 |            52797 |
+|        66 | t_64_r_31766 | t_64_r_63531 |        67 |               1 | 67    |          0 |          1325 |   72522521 |                  104 |            78495 |
+|        68 | t_64_r_63531 | t_64_r_95296 |        69 |               1 | 69    |          0 |          1325 |          0 |                  104 |            95433 |
+|         2 | t_64_r_95296 |              |         3 |               1 | 3     |          0 |          1501 |          0 |                   81 |            63211 |
++-----------+--------------+--------------+-----------+-----------------+-------+------------+---------------+------------+----------------------+------------------+
+4 rows in set (0.00 sec)
+```
+
+Using `EXPLAIN` with the following aggregation statement, we can see that `└─StreamAgg_8` is first performed on each region inside TiKV. Each TiKV region will then send one row back to TiDB, which aggregates the data from each region in `StreamAgg_16`:
+
+{{< copyable "sql" >}}
+
+```sql
+EXPLAIN SELECT COUNT(*) FROM t1;
+```
+
+```sql
++----------------------------+-----------+-----------+---------------+---------------------------------+
+| id                         | estRows   | task      | access object | operator info                   |
++----------------------------+-----------+-----------+---------------+---------------------------------+
+| StreamAgg_16               | 1.00      | root      |               | funcs:count(Column#7)->Column#5 |
+| └─TableReader_17           | 1.00      | root      |               | data:StreamAgg_8                |
+|   └─StreamAgg_8            | 1.00      | cop[tikv] |               | funcs:count(1)->Column#7        |
+|     └─TableFullScan_15     | 242020.00 | cop[tikv] | table:t1      | keep order:false                |
++----------------------------+-----------+-----------+---------------+---------------------------------+
+4 rows in set (0.00 sec)
+```
+
+This is easiest to observe in `EXPLAIN ANALYZE`, where the `actRows` matches the number of regions from `SHOW TABLE REGIONS` because a `TableFullScan` is being used and there are no secondary indexes:
```

Comment:
```suggestion
This is easiest to observe in `EXPLAIN ANALYZE`, where the `actRows` matches the number of Regions from `SHOW TABLE REGIONS` because a `TableFullScan` is being used and there are no secondary indexes:
```